lib/sqlalchemy/dialects/postgresql/ranges.py

```diff
@@ -80,6 +80,87 @@ def __init__(
     def __bool__(self) -> bool:
         return self.empty
 
+    def contains_value(self, value: _T) -> bool:
+        "Check whether this range contains the given `value`."
+
+        if self.empty:
+            return False
+
+        if self.lower is None:
+            if self.upper is None:
+                return True
+            return self.upper is None or (
```

Comment:
`self.upper is None` is always true on this line

Reply:
Right, nice spot.
Now that this is "carried" on Gerrit, can I keep going pushing new changesets here or what?

Reply:
Sure, continue here, we can ask the bot to update gerrit from the pr